src/main/java/org/opentestsystem/ap/imrt/common/model/ImrtItem.java

```diff
@@ -1,135 +1,31 @@
-import org.opentestsystem.ap.common.model.Item;
-import org.springframework.data.jpa.domain.support.AuditingEntityListener;
-
-import javax.persistence.Basic;
-import javax.persistence.Column;
+import javax.persistence.CascadeType;
+import javax.persistence.DiscriminatorValue;
 import javax.persistence.Entity;
-import javax.persistence.EntityListeners;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
-import javax.validation.constraints.NotNull;
-import java.time.Instant;
+import java.util.List;
 
 @Entity
 @Table(name = "item")
-@EntityListeners(AuditingEntityListener.class)
-public class ImrtItem extends BaseEntity {
-
-    @Id
-    @Column
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Integer key;
-
-    @NotBlank
-    private String id;
-
-    private String subject = "";
-
-    private String grade = "";
-
-    @NotBlank
-    private String workflowStatus;
-
-    @NotBlank
-    private String itemType;
-
-    @Column(name = "dok")
-    private String depthOfKnowledge = "";
-
-    @NotBlank
-    private String itemCreatedBy;
-
-    @Basic
-    private Instant itemCreatedAt;
-
-    @Type(type = "jsonb")
-    @Column(columnDefinition = "jsonb")
-    @NotNull
-    private Item itemJson;
-
-    public Integer getKey() {
-        return key;
-    }
-
-    public void setKey(Integer key) {
-        this.key = key;
-    }
-
-    public String getId() {
-        return id;
-    }
-
-    public void setId(String id) {
-        this.id = id;
-    }
-
-    public String getSubject() {
-        return subject;
-    }
-
-    public void setSubject(String subject) {
-        this.subject = subject;
-    }
-
-    public String getGrade() {
-        return grade;
-    }
-
-    public void setGrade(String grade) {
-        this.grade = grade;
-    }
-
-    public String getWorkflowStatus() {
-        return workflowStatus;
-    }
-
-    public void setWorkflowStatus(String workflowStatus) {
-        this.workflowStatus = workflowStatus;
-    }
-
-    public String getItemType() {
-        return itemType;
-    }
-
-    public void setItemType(String itemType) {
-        this.itemType = itemType;
-    }
-
-    public String getDepthOfKnowledge() {
-        return depthOfKnowledge;
-    }
-
-    public void setDepthOfKnowledge(String depthOfKnowledge) {
-        this.depthOfKnowledge = depthOfKnowledge;
-    }
-
-    public String getItemCreatedBy() {
-        return itemCreatedBy;
-    }
-
-    public void setItemCreatedBy(String itemCreatedBy) {
-        this.itemCreatedBy = itemCreatedBy;
-    }
-
-    public Instant getItemCreatedAt() {
-        return itemCreatedAt;
-    }
+@DiscriminatorValue("item")
+public class ImrtItem extends BaseItem {
+    @JoinColumn(name = "item_key")
+    @OneToMany(
```

Comment:
I need to redo stimlink to not have the composite key.  Any time I try to create a OneToOne JPA gets angry because Stimlink has the composite key within the code.